content/configuration/dev-server.md

```diff
@@ -403,6 +566,11 @@ For example, the dev-server is proxied by nginx, and available on `myapp.test`:
 public: "myapp.test:80"
 ```
 
+Usage via the CLI
+```
+webpack-dev-server --public myapp.test:80
+```
+
 
 ## `devServer.publicPath` 🔑
```

Comment:
@simon04 (or @orteth01) do either of you know the purpose of the key icons? Can we drop them?